### DISCUSSION TITLE
Field containing data?

### DISCUSSION THREAD

**Discussion by Nehrert** [farling42/obsidian-import-json#42] [Q&A] [answered]
I've exported my DayOne journal to JSON and I want to import it to Obsidian. I'm not a coder, just a happy enthusiast, so I reach out to you all for help. I have no problem creating a simple template for the import however I came across something that I do no know how to manage.

The journal entries contain images and every image has a child value `identifier`. As you can see the JSON section `photos`contains two values named `identifier`. One is the location data and one is the actual file name of an image stored locally in another folder in my vault.

How to I make the handlebar template to pick out the identifier for the image name?
 
The JSON code looks like this:

`"photos" : [
    {
      "fileSize" : 1273508,
      "location" : {
        "region" : {
          "center" : {
            "longitude" : 16.539303100000012,
            "latitude" : 59.608956697002284
          },
          "identifier" : "<+59.60895670,+16.53930310> radius 70.78",
          "radius" : 70.778074713358052
        },
        "localityName" : "Västerås",
        "country" : "Sverige",
        "timeZoneName" : "Europe\/Stockholm",
        "administrativeArea" : "Västmanlands län",
        "longitude" : 16.539421081542969,
        "placeName" : "Övre Kungsgatan 8",
        "latitude" : 59.609024047851562
      },
      "orderInEntry" : 0,
      "duration" : 0,
      "favorite" : false,
      "type" : "jpeg",
      "identifier" : "A8E50189305943C289E7C170FCC646CB",
      "height" : 2100,
      "fnumber" : "(null)",
      "width" : 1575,
      "md5" : "89b80c0f2e72e66c0cd48af5a43c06b5",
      "isSketch" : false,
      "focalLength" : "(null)"
    }
  ],`

**Comment by farling42** [comment, marked as answer]
If you format it in a nice way, then you see that "identifier" can be accessed by two different paths:
```
{
	"photos": [{
		"fileSize": 1273508,
		"location": {
			"region": {
				"center": {
					"longitude": 16.539303100000012,
					"latitude": 59.608956697002284
				},
				"identifier": "<+59.60895670,+16.53930310> radius 70.78",
				"radius": 70.778074713358052
			},
			"localityName": "Västerås",
			"country": "Sverige",
			"timeZoneName": "Europe\/Stockholm",
			"administrativeArea": "Västmanlands län",
			"longitude": 16.539421081542969,
			"placeName": "Övre Kungsgatan 8",
			"latitude": 59.609024047851562
		},
		"orderInEntry": 0,
		"duration": 0,
		"favorite": false,
		"type": "jpeg",
		"identifier": "A8E50189305943C289E7C170FCC646CB",
		"height": 2100,
		"fnumber": "(null)",
		"width": 1575,
		"md5": "89b80c0f2e72e66c0cd48af5a43c06b5",
		"isSketch": false,
		"focalLength": "(null)"
	}]
}
```
so either just `identifier` to get the alphanumeric string, or `location.region.identifier` for the value that includes the lat-long and radius.

You might need to use `{{#each photos}}` to loop through each entry in the photos array.

**Reply by Nehrert** [reply]
Perfect. That syntax made it all clear. (the dot between the values)
